### ISSUE TITLE
[Docs]: 

### ISSUE THREAD

**Issue by jvaldesuso** [hashicorp/terraform-provider-aws#32600]
### Documentation Link

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl_logging_configuration

### Description

There is an error in the data policy document, specifically with the principals block. `type` should be `Service` and not `AWS`

### Would you like to implement a fix?

None

**Comment by bebold-jhr** [comment]
@jvaldesuso Looks like this has been fixed already. The [example](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/wafv2_web_acl_logging_configuration#with-cloudwatch-log-group-and-managed-cloudwatch-log-resource-policy) shows principal "service".

```hcl
principals {
      identifiers = ["delivery.logs.amazonaws.com"]
      type        = "Service"
    }
```

**Comment by justinretzolk** [comment]
Hey @jvaldesuso 👋 Thank you for taking the time to raise this! As noted above, this was fixed with #32570, and is now correct in the Registry documentation. We'll go ahead and close this issue out with that in mind, but really appreciate you taking the time to report this!
